src/test/java/net/openhft/chronicle/wire/UnsupportedChangesTest.java

```diff
@@ -19,39 +19,27 @@
 package net.openhft.chronicle.wire;
 
 import net.openhft.chronicle.core.Jvm;
-import net.openhft.chronicle.core.onoes.ExceptionKey;
-import net.openhft.chronicle.core.onoes.LogLevel;
 import org.junit.Test;
 
-import java.util.Collections;
-import java.util.Map;
-
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertNotNull;
 import static org.junit.Assume.assumeFalse;
 
 public class UnsupportedChangesTest extends WireTestCommon {
     @Test
     public void scalarToMarshallable() {
-        Map<ExceptionKey, Integer> exceptions = Jvm.recordExceptions(true);
-
         Nested nested = Marshallable.fromString(Nested.class, "{\n" +
                 "inner: 128\n" +
                 "}\n");
         assertEquals("!net.openhft.chronicle.wire.UnsupportedChangesTest$Nested {\n" +
                 "  inner: !!null \"\"\n" +
                 "}\n", nested.toString());
-        ExceptionKey ek = new ExceptionKey(LogLevel.WARN,
-                WireMarshaller.ObjectFieldAccess.class,
-                "Unable to parse field: inner, as a marshallable as it is 128",
-                exceptions.keySet().iterator().next().throwable);
-        assertEquals(Collections.singletonMap(ek, 1), exceptions);
+        expectException("Unable to parse field: inner, as a marshallable as it is 128");
```

Comment:
These blew up with the new ExceptionKey, because it had an additional field. expectException should be less brittle